scripts/test_nccl.sh

```
@@ -0,0 +1,90 @@
+echo "=== NCCL Smoke Test ==="
+echo "Host: $(hostname) | Date: $(date)"
+echo "CUDA_VISIBLE_DEVICES=$CUDA_VISIBLE_DEVICES"
+
+export MASTER_ADDR=$(hostname)
+export MASTER_PORT=29500
+export NCCL_P2P_DISABLE=1
+export NCCL_SHM_USE_CUDA_MEMCPY=1
+
+UVENV="$HOME/eq_stuff/eq"
+
+srun $UVENV/bin/python -c "
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)